tox.ini

```diff
@@ -1,18 +1,18 @@
 [gh-actions]
 python =
-    3.10: py310
-    3.11: py311,integration
+    3.10: py310,noextras
+    3.11: py311,noextras,integration
 
 [tox]
-envlist = pylint,mypy,pyright,py310,py311,integration,coverage
+envlist = pylint,mypy,pyright,py310,py311,noextras,integration,coverage
 isolated_build = true
 
 [testenv]
 deps =
   -r{toxinidir}/dev.requirements.txt
 
 commands =
-    coverage run -p -m pytest {posargs}
+    coverage run -p -m pytest --ignore tests/unit/test_init_plugin_no_extras.py {posargs}
 
 [testenv:coverage]
 depends = py310,py311
```

Comment:
The coverage env should depend on `noextras` also so we get cover for all the conditional code paths when the extras aren't available, do you agree?

Reply:
Sure